sycl/test-e2e/Basic/image/image_accessor_readsampler.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: aspect-ext_intel_legacy_image
-// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: cuda || target-amd
```

Comment:
This is a bit surprising that it's a runtime condition for CUDA and compile-time for AMD. tagging @intel/llvm-reviewers-cuda for any potential follow-up to this.